lib/interfaces/include/WatchdogInterface.h

```diff
@@ -0,0 +1,47 @@
+#ifndef WATCHDOG_INTERFACE_H
+#define WATCHDOG_INTERFACE_H
+
+#include <etl/singleton.h>
+#include <Arduino.h>
+
+using pin = size_t;
+
+/* Watchdog Constants */
+namespace Watchdog_Params {
+
+    constexpr const pin WATCHDOG_PIN = 26; //teensy input to watchdog
```

Comment:
you don't need to put the "using" statement for pin, you can just declare as int or unsigned int